.vscode/launch.json

```diff
@@ -29,10 +29,10 @@
             "preLaunchTask": "build",
             // Add ids and secrets here to use it when debuging KeyVaultPOC
             "env": {
-                "tenant_id": "",
-                "client_id": "",
-                "client_secret": "",
-                "test_uri": ""
+                "tenant_id": "72f988bf-86f1-41af-91ab-2d7cd011db47",
```

Comment:
You need to rebase this out of the repo.  Also you need to rotate the secrets.

Reply:
ups, didn't want to push that change,, removed it now